tests/unit/admin/views/test_banners.py

```diff
@@ -215,6 +217,14 @@ def test_required_fields(self):
     def test_valid_data(self):
         form = views.BannerForm(data=self.data)
         assert form.validate() is True
+        data = form.data
+        assert data["name"] == self.data["name"]
+        assert data["text"] == self.data["text"]
+        assert data["link_url"] == self.data["link_url"]
+        assert data["link_label"] == self.data["link_label"]
+        assert data["begin"] == self.data["begin"]
+        assert data["end"] == self.data["end"]
+        assert data["fa_icon"] == Banner.DEFAULT_FA_ICON
```

Comment:
```
assert data == {**self.data, **{"fa_icon": Banner.DEFAULT_FA_ICON}}
```
This will also ensure there's no unexpected additional field.

Reply:
Implemented at 7b74938126e7af575361bac53dea5c1b4f72e506